llvm/lib/CodeGen/ReplaceWithVeclib.cpp

```diff
@@ -100,26 +100,26 @@ static void replaceWithTLIFunction(Instruction &I, VFInfo &Info,
 /// works when \p I is a call to vectorized intrinsic or the frem instruction.
 static bool replaceWithCallToVeclib(const TargetLibraryInfo &TLI,
                                     Instruction &I) {
+  auto *VTy = dyn_cast<VectorType>(I.getType());
+  ElementCount EC(VTy ? VTy->getElementCount() : ElementCount::getFixed(0));
```

Comment:
nit: might be worth to drop a short comment:
"A the moment VFABI assumes the return type is always widened unless it is a void type."